Packs/CommonScripts/ReleaseNotes/1_12_9.json

```diff
@@ -0,0 +1,4 @@
+{
+    "breakingChanges": true,
+    "breakingChangesNotes": null
```

Comment:
add the change notes, see [this](https://github.com/demisto/content/blob/master/Packs/EDL/ReleaseNotes/3_1_26.json) for example